frontend/hub/HubMasthead.tsx

```
@@ -91,3 +98,40 @@ export function HubMasthead() {
     </PageMasthead>
   );
 }
+
+export function useHubNotifications() {
+  const { t } = useTranslation();
+  const getPageUrl = useGetPageUrl();
+
+  const { hasPermission } = useHubContext();
+
+  const canApprove = hasPermission('ansible.modify_ansible_repo_content');
+
+  const { data: result } = useGet<HubItemsResponse<CollectionVersionSearch>>(
+    canApprove ? hubAPI`/v3/plugin/ansible/search/collection-versions/` : undefined,
+    { page_size: 200, repository_label: 'pipeline=staging' },
```

(max page_size is 100, but the api will silently ignore this)